### DISCUSSION TITLE
https://react-hook-form.com/ down?

### DISCUSSION THREAD

**Discussion by DirkWolthuis** [react-hook-form/react-hook-form#4160] [General] [answered]
Hi all, is the doc website down?

**Comment by bluebill1049** [comment]
it's up for me.

**Comment by bluebill1049** [comment, marked as answer]
<img width="363" alt="Screen Shot 2021-02-11 at 8 10 53 pm" src="https://user-images.githubusercontent.com/10513364/107618179-40f2f500-6ca5-11eb-9d57-f432d2148074.png">

**Reply by DirkWolthuis** [reply]
haha im sorry it our company VPN guess we can't use your lib anymore :P
